src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerDraftOrder.java

```diff
@@ -71,7 +72,7 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
     public SubmittedCallbackResponse draftCreated(CaseDetails<CaseData, State> details,
                                                   CaseDetails<CaseData, State> beforeDetails) {
         return SubmittedCallbackResponse.builder()
-            .confirmationHeader("Draft order created")
+            .confirmationHeader(format("Draft order created"))
```

Comment:
Is this format needed?

Reply:
no this does not need. this has been removed